### ISSUE TITLE
Error in nativecheckout v2 - PaymentsCardHolderAuthenticationInvalidStatus

### ISSUE THREAD

**Issue by castillo2310** [VivaPayments/API#2114]
Hello,

I'm integrating Native checkout v2 and I'm having problems with the POST https://demo-api.vivapayments.com/nativecheckout/v2/transactions request.
I always get the following response:
{"status":400,"message":"PaymentsCardHolderAuthenticationInvalidStatus","eventId":"2061"}

Any advice would be appreciated. I'm using the card 5188340000000011 from the developer documents.

Thanks!

**Comment by matthewrgourd** [comment]
Hi @castillo2310, very strange, I just tried myself using the Working example in HTML to generate the charge token (appending the code first with my bearer token), then made the below call and got a transaction ID successfully:

POST `https://demo-api.vivapayments.com/nativecheckout/v2/transactions`

REQUEST BODY
```
{
  "amount": 550,
  "chargeToken": "ctok_Pym9_d7MXyn6OF0RvZuALj1ISvM",
  "merchantTrns": "Merchant transaction reference",
  "customerTrns": "Description that the customer sees",
  "currencyCode": 826,
  "customer": {
      "email": "mgourd@vivawallet.com",
      "phone": "447498719486",
      "fullname": "Joe Blogs",
      "requestLang": "en",
      "countryCode": "GB"
   }
}
```
RESPONSE
```
{
    "transactionId": "73bb7b91-808a-47e7-a99a-2e4d676c8d3b",
    "redirectUrl": null
}
```

However, the next time I tried, I got the same as you. So I think there are intermittent problems on demo with the card number 5188340000000011. Please can you try 5239290700000101 instead and let me know how you get on. Thanks!

**Comment by castillo2310** [comment]
Hello @matthewrgourd, I'm trying with the other card, but I'm getting a 403 Forbidden response.

The request data is the following:
```json
{
  "amount": 10.0,
  "preauth": false,
  "sourceCode": "3212",
  "chargeToken": "ctok_8paKJfzU_ylF2WsK0JI3QvLEJM0",
  "installments": 1,
  "merchantTrns": "1020210126183531",
  "customerTrns": "Descripcion pago",
  "currencyCode": 978,
  "customer": {
    "email": "castillo.2310@gmail.com",
    "phone": "680123456",
    "fullname": "Cristian Castillo",
    "countryCode": "ES",
    "requestLang": "es"
  }
}

**Comment by matthewrgourd** [comment]
Hi @castillo2310, 403 forbidden relates to your authentication. Please go to your Viva Wallet account and generate a new token:

![Screenshot_2021-01-27_at_08_30_32](https://user-images.githubusercontent.com/44943019/105964375-00c32e00-607a-11eb-96e4-b65648eeb607.png)

...then follow our instructions for [requesting an access token](https://developer.vivawallet.com/web-api-integration/authentication/#step-2-request-access-token).

**Comment by castillo2310** [comment]
Hello @matthewrgourd, thanks for your response. I generated a new clientId and with the card 5239290700000101 and I'm still getting a 403 response, but this time with this json {"status":403,"message":"Invalid input","eventId":"1"}.

I don't know what I'm doing wrong.
First I generate and access token.
Second i use the access token in the client side payment form to get a charge token.
Finally, with the access token in the header, and the trasaction data I try to do the POST request to https://demo-api.vivapayments.com/nativecheckout/v2/transactions.

I've attached two screenshots from postman with the headers, body and response, because I'm following the guide and I don't see what is wrong.

![image](https://user-images.githubusercontent.com/5794177/106030184-32fe7b00-60ce-11eb-969b-3fe64b16d5e9.png)
![image](https://user-images.githubusercontent.com/5794177/106030248-47db0e80-60ce-11eb-96e7-4bdb4088a900.png)

![image](https://user-images.githubusercontent.com/5794177/106030126-1eba7e00-60ce-11eb-8fc5-718a4a53d656.png)
